config/locales/metadata/en.yml

```diff
@@ -496,7 +496,7 @@ en:
           ebi_db: ENA-STUDY
 
         hmdmc_approval_number:
```

Comment:
shouldn't this be renamed if it's now labelled HuMFre? hmdmc_approval_number

Reply:
No, the same field name is used for database columns and form fields. It is the same approval number with some change in the meaning. We just need to change the label.